physionet-django/user/management/commands/reject_pending_credentialing_applications.py

```diff
@@ -0,0 +1,70 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.db import transaction
+from django.http import HttpRequest
+from django.utils import timezone
+
+import notification.utility as notification
+from user.models import CredentialApplication, User
+
+LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NUMBER_OF_APPLICATIONS_TO_REJECT = 5
```

Comment:
Is there a reason why we don't treat this as.a configurable variable in the .env file?

Reply:
Thats an excellent point, making it configurable makes a lot of sense. Let me do that.